src/plugins/collapsibleColumns/test/collapsibleColumns.e2e.js

```diff
@@ -305,5 +305,35 @@ describe('CollapsibleColumns', function() {
 
     });
 
+    it('should expand to master table width after editing and clicking the expand button #105', function() {
+      const hot = handsontable({
+        data: Handsontable.helper.createSpreadsheetData(10, 10),
+        rowHeaders: true,
+        hiddenColumns: true,
+        nestedHeaders: [
+          ['a', {label: 'd', colspan: 4}, 'c', 'd', 'e', 'f', 'g', 'a', 'b', 'c', 'd', 'e', 'f', 'g'],
+          ['a', {label: 'b', colspan: 2}, {label: 'c', colspan: 2}, 'd', 'e', 'a', 'b', 'c', 'd', 'e', 'f', 'g']
+        ],
+        collapsibleColumns: true
+      });
+      let topCloneHiderWidth = document.querySelector('.ht_clone_top .wtHider').style.width;
+
+      const button = $('.collapsibleIndicator').first();
+
+      button.simulate('mousedown');
+
+      expect(document.querySelector('.ht_clone_top .wtHider').style.width).not.toBe(topCloneHiderWidth);
```

Comment:
I think the point should be to verify the fact that master's and clone_top's `.wtHider`s have the same width.